pkg/accounting/accounting.go

```diff
@@ -1159,7 +1159,7 @@ func (a *Accounting) NotifyRefreshmentSent(peer swarm.Address, attemptedAmount,
 	// compare received refreshment amount to expectation
 	if expectedAllowance.Cmp(amount) > 0 {
 		// if expectation is not met, blocklist peer
-		a.logger.Error(ErrEnforceRefresh, "pseudosettle peer", peer, "accepted lower payment than expected")
+		a.logger.Error(ErrEnforceRefresh, "accepted lower payment than expected", "pseudosettle peer", peer.String())
```

Comment:
Unnecessary to call `.String()`. The error should also be `nil` instead of `ErrEnforceRefresh` because the caller did not return any error.